index.js

```diff
@@ -78,6 +78,15 @@ var reqTask = function (task) {
 
 var projPkg = grunt.file.readJSON('package.json')
 grunt.initConfig({
+  ci: {
+    buildNumber: process.env.BUILD_NUMBER || 0,
+    buildId: process.env.BUILD_ID || 0,
+    buildUrl: process.env.BUILD_URL || "",
+    buildTag: process.env.BUILD_TAG || "",
+    buildCommit: (process.env.GIT_COMMIT !== "" && process.env.GIT_COMMIT !== undefined) ? process.env.GIT_COMMIT : "",
+    buildCommitShort: (process.env.GIT_COMMIT !== "" && process.env.GIT_COMMIT !== undefined) ? process.env.GIT_COMMIT.substring(0, 7) : "",
+    warNameSuffix: (process.env.BUILD_NUMBER !== undefined) ? "-" + process.env.BUILD_NUMBER : ""
```

Comment:
If the build number isn't used for local scenarios, what if we created the whole string here '=BUILD###' when the number is defined and empty when it isn't so that we just get a clean package version locally?

Reply:
We could do that, but it was including the `-BUILD` part before I came along so I didn't want to change that in case someone/thing is depending on that.

Reply:
That makes sense - that didn't occur to me.